dao/classes/AssignmentDAO.java

```diff
@@ -0,0 +1,121 @@
+package com.solvd.db.mysql.dao.classes;
+
+import com.solvd.db.mysql.dao.AbstractDAO;
+import com.solvd.db.mysql.dao.IDAO;
+import com.solvd.db.mysql.model.Assignment;
+import com.solvd.db.mysql.model.ClassTable;
+import com.solvd.db.utils.ConnectionManager;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AssignmentDAO extends AbstractDAO<Assignment> implements IDAO<Assignment> {
+    private static final Logger logger = LogManager.getLogger(StudentDAO.class);
+    public static final String insertQuery = "insert into assignments(name, due_date, score, class_id) " +
+            "values(?,?,?,?)";
+    public static final String updateQuery = "update assignments set name = ?, due_date = ?, score = ?, class_id = ? where id = ?";
```

Comment:
It was made according to Vladislav's suggestion to keep queries as instance variables and this will give relief if I need to use them somewhere else..